setup.py

```diff
@@ -90,7 +90,7 @@ def run(self):
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
     ],
-    packages=find_packages(exclude=["tests"]),
+    packages=find_packages(exclude=["tests", "tests.*"]),
```

Comment:
@pwoolvett, do you know why the `tests` didn't work? The examples on the [`find_packages`](https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html?highlight=exclude#using-find-or-find-packages) looks similar to what we have.

Reply:
in the examples, `additional` is a package without subpackages
in `dvc`'s `tests`, there are subpackages (eg `unit`

```bash
$ tree
.
├── demo
│   ├── additional
│   │   ├── __init__.py
│   │   └── not_included
│   │       └── __init__.py
│   └── __init__.py
├── setup.py
└── tests
    ├── __init__.py
    └── unit
        └── __init__.py

5 directories, 6 files

$ python -c "from setuptools import find_packages;print(find_packages(exclude=['additional']))"
['demo', 'tests', 'demo.additional', 'demo.additional.not_included', 'tests.unit']

$ python -c "from setuptools import find_packages;print(find_packages(exclude=['additional', 'additional.*']))"

['demo', 'tests', 'demo.additional', 'demo.additional.not_included', 'tests.unit']

$ python -c "from setuptools import find_packages;print(find_packages(include=['demo', 'demo.*']))"
['demo', 'demo.additional', 'demo.additional.not_included']
```

Reply:
I guess it uses a wild match algorithm and treats all of the packages as strings.

 `demo*` can match all of the packages in `['demo', 'demo.sub', 'demo.sub2', 'demo.sub.inside']`

```shell
$ python -c "from setuptools import find_packages;print(find_packages(exclude=['demo']))"
[]
$ tree demo
demo
├── __init__.py
├── sub
│   ├── __init__.py
│   └── inside
│       └── __init__.py
└── sub2
    └── __init__.py
$ python -c "from setuptools import find_packages;print(find_packages(exclude=['']))"

['demo', 'demo.sub', 'demo.sub2', 'demo.sub.inside']
```

So I guess in this case `tests*` can replace the `"tests", "tests.*"` but as it can also match some other packages like `testa` or `testb`, it is better not to do it.

Reply:
Could have also used `include=['dvc']`, but this seemed the least invasive...